src/Database/Bloodhound/Types/Class.hs

```diff
@@ -3,9 +3,9 @@ module Database.Bloodhound.Types.Class
        ( Seminearring(..) )
        where
 
-#if !MIN_VERSION_base(4,8,0)
+-- #if !MIN_VERSION_base(4,8,0)
```

Comment:
Whoops. My bad. I had done that on a different day to get haskell interactive process and stack to play nice and forgot to put it back.